package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@launchpadlab/lp-subsection-generator",
-  "version": "6.1.0",
+  "version": "7.0.0",
```

Comment:
MAJOR REV!!! See my note below.

Reply:
Can you add a) a migration guide and b) a pre-reqs section (e.g., https://github.com/LaunchPadLab/client-template#prerequisites)?

Additionally, do we need to add an `"engines"` declaration to this file?

Reply:
Good ideas. I think the "engines" section will help installation dependency problems show up more explicitly. I'll set to >= Node 12 for now.